src/main/java/menu/controller/ServiceManageController.java

```diff
@@ -0,0 +1,73 @@
+package menu.controller;
+
+import menu.domain.Coach;
+import menu.domain.CoachInfo;
+import menu.domain.ExceptMenu;
+import menu.service.RecommendService;
+import menu.view.InputView;
+import menu.view.OutputView;
+
+import java.util.List;
+
+public class ServiceManageController {
+
+    private final InputView inputView;
+    private final OutputView outputView;
+    private final RecommendService recommendService;
+    private CoachInfo coachInfo;
+
+    public ServiceManageController() {
+        inputView = new InputView();
+        outputView = new OutputView();
+        recommendService = new RecommendService();
+    }
+
+    public void startRecommend() {
+        outputView.introMessage();
+        Coach coach = getCoachName();
+        for(String coachName : coach.getCoachNames()) {
+            coachInfo = getCoachInfo(coachName);
+        }
+        recommendService.recommendMenu(coachInfo);
+        getRecommendResult(coachInfo);
+    }
+
+    private Coach getCoachName() {
+        outputView.coachNameMessage();
+        try {
+            return new Coach(inputView.getInput());
+        } catch (IllegalArgumentException exception) {
+            outputView.errorMessage();
+            return getCoachName();
+        }
+    }
+
+    private ExceptMenu getExceptMenu(String coachName) {
+        try {
+            outputView.exceptMenuMessage(coachName);
+            return new ExceptMenu(inputView.getInput());
+        } catch (IllegalArgumentException exception) {
+            outputView.errorMessage();
+            return getExceptMenu(coachName);
+        }
+    }
+
+    private CoachInfo getCoachInfo(String coachName) {
+        ExceptMenu exceptMenu = getExceptMenu(coachName);
+        List<String> exceptMenus = exceptMenu.getExceptMenus();
+        coachInfo = setCoachInfo(coachName, exceptMenus);
+
+        return coachInfo;
+    }
+
+    private CoachInfo setCoachInfo(String coachName, List<String> exceptMenus) {
+        return new CoachInfo(coachName, exceptMenus);
+    }
```

Comment:
```suggestion
    private CoachInfo getCoachInfo(String coachName) {
        ExceptMenu exceptMenu = getExceptMenu(coachName);
        List<String> exceptMenus = exceptMenu.getExceptMenus();

        return new CoachInfo(coachName, exceptMenus);
    }
```
이게 더 깔끔할 것 같은데 어떻게 생각하시나요?